.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "tabWidth": 2
```

Comment:
good idea to add the `.prettierrc` 👍

we use empty `{}` in other nns related projects, can you adapt?